src/Microsoft.AspNetCore.Antiforgery/AntiforgeryTokenSet.cs

```diff
@@ -23,22 +23,13 @@ public class AntiforgeryTokenSet
             string formFieldName,
             string headerName)
         {
-            if (string.IsNullOrEmpty(requestToken))
-            {
-                throw new ArgumentException(Resources.ArgumentCannotBeNullOrEmpty, nameof(requestToken));
-            }
```

Comment:
Previously we had an invariant that the `AntiforgeryTokenSet` could not be invalid. Probably :ok: as long as `DefaultAntiforgery` can never return such a token set. But have you confirmed we're covered?